python/mxnet/initializer.py

```diff
@@ -431,8 +432,32 @@ def _init_weight(self, _, arr):
 
 
 @register
+class LSTMCellForgetBias(Initializer):
```

Comment:
Consider dropping "ForgetBias" from the name in case other LSTM param initialization want to reuse this.